src/examples/radiuss_tutorial/host-config.cmake.in

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+# other Axom Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+#------------------------------------------------------------------------------
+# Host-config for 'radiuss_tutorial' example. 
+# Defines configuration variables from Axom's build.
+#------------------------------------------------------------------------------
+
+# Config related to compiler
+set(CMAKE_C_COMPILER              "@CMAKE_C_COMPILER@"       CACHE PATH "")
+set(CMAKE_CXX_COMPILER            "@CMAKE_CXX_COMPILER@"     CACHE PATH "")
+set(CMAKE_Fortran_COMPILER        "@CMAKE_Fortran_COMPILER@" CACHE PATH "")
```

Comment:
Is fortran being used in this tutorial?

Reply:
No -- I adapted this from the `using_with_blt` one. I'm pretty sure our TPLs were also built w/ fortran in the docker image